### DISCUSSION TITLE
[Proposal] Show a popup over another popup on Mac

### DISCUSSION THREAD

**Discussion by gauravK996** [CommunityToolkit/Maui#2131] [New Feature Discussions]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

Currently, there is an issue preventing the opening of a popup on top of another popup on Mac.

### Expected Behavior

The expected behavior is that when attempting to open a popup over another popup on Mac, it should open seamlessly with a proper anchor. The topmost popup should display as intended without any visual glitches or interference from the underlying popup.

### Steps To Reproduce

**Steps to Reproduce:**
1. Launch the application on a Mac system.
2. Trigger the first popup by clicking on a button.
3. Attempt to open a second popup over first popup while the first popup is still visible.
4. Observe the behavior on Mac. 

### Link to public reproduction project repository

https://github.com/gauravK996/MauiApp4

### Environment

```markdown
- .NET MAUI CommunityToolkit:7.0.1
- OS: 14.3.1 Sonoma
- .NET MAUI: .NET MAUI 8 (8.0.6)
- Xcode version 15.2
```

### Anything else?

Last version that worked well
.NET MAUI 7.0.101
.NET MAUI CommunityToolkit 5.3.0

**Comment by hirendh** [comment]
I am facing the same issue opening of a popup on top of another popup on Mac. 

My configuration as below :

> - .NET MAUI CommunityToolkit:7.0.1
> - OS: 14.3.1 Sonoma
> - .NET MAUI: .NET MAUI 8 (8.0.6)

**Comment by vhugogarcia** [comment]
In Mac Catalyst, displaying one popover over another can be tricky due to platform limitations. Mac Catalyst is designed to bridge iOS apps to macOS, but it doesn’t always support all iOS features seamlessly.

However, you can work around this limitation by managing the presentation of your popovers carefully. One approach is to dismiss the first popover before presenting the second one. Alternatively, you can use a custom view controller to manage multiple popovers, ensuring they don’t overlap in a way that causes issues. On this second option, it will be mostly a feature requested, and not a bug, since currently multiple popovers is not supported in Mac Catalyst. 

I'm moving this to a discussion for further feedback and vote for this feature.
